README.md

```diff
@@ -16,6 +16,10 @@ SiteFi is a simple and highly configurable static site generator for F#. It uses
  * Develop dynamic articles in F#, with charts, visualizations, etc.
  * Streamlined workflow for template changes (style, layout, etc.) - see effects immediately, and only rebuild when you are done
 
+## Introduction 
+
+See my [blog](https://www.intellifactory.com/blog/5862/f-metablogging-introducing-blogengine-for-your-static-markdown-based-f-blog) article about Sci-Fi, that I called BlogEngine in that time, in order to get an overview and extensive tutorial.
```

Comment:
You misspelled SiteFi ;) How about that fragment being "... article about SiteFi, which I called BlogEngine at that time, in order to ..."?

Reply:
See my blog article, it is supposed to mean.
Its no fragment anymore, once its merged and displayed by the markup, only the url braces let it seem like that.